org.gabmus.whatip.json

```diff
@@ -0,0 +1,339 @@
+{
+    "app-id" : "org.gabmus.whatip",
+    "command" : "whatip",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.34",
+    "sdk" : "org.gnome.Sdk",
+    "finish-args" : [
+    	"--device=dri",
+        "--share=ipc",
+        "--socket=x11",
+        "--share=network",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--filesystem=xdg-run/dconf",
```

Comment:
these dconf permissions are now built in unless you need something special from the system integration and should be removed.